packages/block-library/src/image/edit.native.js

```diff
@@ -288,11 +278,11 @@ class ImageEdit extends React.Component {
 					{ ( ! this.state.isCaptionSelected ) &&
 						getToolbarEditButton( openMediaOptions )
 					}
-					<InspectorControls>
+					<InspectorControls buttons>
 						<ToolbarButton
 							title={ __( 'Image Settings' ) }
 							icon="admin-generic"
-							onClick={ onImageSettingsButtonPressed }
+							onClick={ onSettingsToggle }
```

Comment:
If I understood correctly you can pass `onSettingsToggle` via fillProps that way you don't need to pass it from component to component. 

I see some example usage in the code base but haven't tested myself:

```
    <__experimentalBlockSettingsMenuFirstItem.Slot
                                        fillProps={ { onClose } }
                           />
```

Passing to the `Fill`:

```
			<__experimentalBlockSettingsMenuFirstItem>
				{ ( { onClose } ) => <BlockInspectorButton onClick={ onClose } /> }
			</__experimentalBlockSettingsMenuFirstItem>
```

Reply:
I have also thought about that and already did some work - will present how it works in next commits.

According to add wrapper for BottomSheet.Cell components - that's fine. We will do it separate PR. I have already posted a PR to add RangeCell implementation in #17282

Reply:
done